src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_AsOperator.cs

```diff
@@ -15,7 +17,7 @@ internal sealed partial class LocalRewriter
         public override BoundNode VisitAsOperator(BoundAsOperator node)
         {
             BoundExpression rewrittenOperand = VisitExpression(node.Operand);
-            var rewrittenTargetType = (BoundTypeExpression)VisitTypeExpression(node.TargetType);
+            var rewrittenTargetType = (BoundTypeExpression)VisitTypeExpression(node.TargetType)!;
```

Comment:
Can `LocalRewriter.VisitTypeExpression` be annotated  to show that it doesn't return null?

Reply:
Yes, but I have to add an override.

---
In reply to: [386595905](https://github.com/dotnet/roslyn/pull/42068#discussion_r386595905) [](ancestors = 386595905)